lambdas/drivingpermitcheck/src/testFixtures/java/uk/gov/di/ipv/cri/drivingpermit/util/HttpResponseFixtures.java

```diff
@@ -0,0 +1,244 @@
+package uk.gov.di.ipv.cri.drivingpermit.util;
+
+import org.apache.http.Header;
+import org.apache.http.HeaderIterator;
+import org.apache.http.HttpEntity;
+import org.apache.http.ProtocolVersion;
+import org.apache.http.StatusLine;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.params.HttpParams;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.Locale;
+
+public class HttpResponseFixtures {
```

Comment:
Does this file not already exist in utils?

Reply:
A similar response fixture was used just for http status codes in DVA/DCS response tests. It has been removed and the tests updated to call this one.